hist/histv7/inc/ROOT/RHist.hxx

```diff
@@ -333,6 +340,81 @@ void Add(RHist<DIMENSIONS, PRECISION, STAT_TO...> &to, const RHist<DIMENSIONS, P
    toImpl.GetStat().Add(fromImpl.GetStat());
 }
 
+/// Divide the `to` histogram by `from`.
```

Comment:
On "to" and "from": divide has "what" and "why what", not I don't see a direction. I understand that you want to convey that "to" is modified. But maybe "hist" and "divisor" are enough?